cmd/podman/restore.go

```diff
@@ -63,6 +61,22 @@ func restoreCmd(c *cliconfig.RestoreValues) error {
 	options := libpod.ContainerCheckpointOptions{
 		Keep:           c.Keep,
 		TCPEstablished: c.TcpEstablished,
+		TargetFile:     c.Import,
+		Name:           c.Name,
+	}
+
+	if c.Import == "" && c.Name != "" {
+		return errors.Errorf("--name can only used with --import")
+	}
+
+	if c.Name != "" && c.TcpEstablished {
+		return errors.Errorf("--tcp-established cannot be used with --name")
+	}
+
+	if c.Import == "" {
+		if err := checkAllAndLatest(cmd, restoreCommand.InputArgs, false); err != nil {
```

Comment:
Alright, I see this was moved here specifically - why? Would it be better to unconditionally check those two where we usually do, then separately check that they conflict with --import?

Reply:
Using `--import` I want to skip the check if `-a`, `-l` or container ID is specified, because with `--import` all the information are part of the import. I thought if I do not do it the way like I did it it is not possible to bypass the `checkAllAndLatest()` from above as it will run unconditionally. At least that was my understanding of cobra. If there is a better way to skip `checkAllAndLatest()` for certain cases I can change it. Right now I do not know how.

Reply:
I would this that `--all` and `--latest` would conflict with `--import`, potentially?